app/services/attachments/create_service.rb

```diff
@@ -26,74 +26,76 @@
 # See docs/COPYRIGHT.rdoc for more details.
 #++
 
-class Attachments::CreateService < ::BaseServices::Create
-  include Attachments::TouchContainer
+module Attachments
+  class CreateService < BaseService
+    include TouchContainer
 
-  around_call :error_wrapped_call
+    around_call :error_wrapped_call
 
-  ##
-  # Create an attachment service bypassing the user-provided whitelist
-  # for internal purposes such as exporting data.
-  #
-  # @param user The user to call the service with
-  # @param whitelist A custom whitelist to validate with, or empty to disable validation
-  #
-  # Warning: When passing an empty whitelist, this results in no validations on the content type taking place.
-  def self.bypass_whitelist(user:, whitelist: [])
-    new(user: user, contract_options: { whitelist: whitelist.map(&:to_s) })
-  end
+    ##
+    # Create an attachment service bypassing the user-provided whitelist
+    # for internal purposes such as exporting data.
+    #
+    # @param user The user to call the service with
+    # @param whitelist A custom whitelist to validate with, or empty to disable validation
+    #
+    # Warning: When passing an empty whitelist, this results in no validations on the content type taking place.
+    def self.bypass_whitelist(user:, whitelist: [])
```

Comment:
@oliverguenther Isn't that method here an exact copy of what is already declared in the `BaseService`? If so, shouldn't we remove it here?

Reply:
It was meant to deleted, probably messed up the commit there